app/helpers/curation_concerns/file_set_helper.rb

```diff
@@ -7,9 +7,12 @@ def parent_path(parent)
     end
   end
 
-  def media_display(file_set, locals = {})
-    render media_display_partial(file_set),
-           locals.merge(file_set: file_set)
+  # REVIEW: Since this media display could theoretically work for
```

Comment:
I think this is fine as-is — pulling out to its own helper module would be a good refactor when there's something else that can use media_display.